Source/Task/AsyncQueue.cpp

```diff
@@ -439,12 +439,14 @@ static void EnsureSharedInitialization()
     BOOL pending;
 
     // Only failure modes for InitOnce are usage errors.
-    NT_VERIFY(InitOnceBeginInitialize(&s_sharedInit, 0, &pending, nullptr));
+    BOOL res = InitOnceBeginInitialize(&s_sharedInit, 0, &pending, nullptr);
+    assert(res);
```

Comment:
>assert(res); [](start = 4, length = 12)

This doesn't generate an unreferenced local warning?

Reply:
libHttpClient is built at warning level 3, and that's a lvl 4 warning